ui/src/app/cohort-search/cohort-search/cohort-search.component.html

```diff
@@ -7,7 +7,7 @@
       </app-search-group-list>
     </div>
 
-    <div class="col-xl-4 col-lg-6" id="exclude-groups">
+    <div *ngIf="(initShowChart$ | async)" class="col-xl-4 col-lg-6" id="exclude-groups">
```

Comment:
Lets put this `*ngIf` direclty on `app-search-group-list` rather than the column div.  And since we have `initShowChart$` here anyways, let's put the same check on the `app-overview` component in this template, and remove it from the `overview` component.  That way this logic stays together at the same level.